app/javascript/src/editable_components.js

```diff
@@ -811,6 +811,7 @@ function sanitiseHtml(html) {
  **/
 function sanitiseMarkdown(markdown) {
   markdown = markdown.replace(/\n<!--.*?-->/mig, "");
+  markdown = markdown.replace(/\]\(\<(.*?)\>\)/mig, "]($1)");
```

Comment:
Can we add a comment here as to what this regex is doing (and why) for future reference, and to the one above would be helpful too.

Reply:
Done.
Commit: abd3ee858202a39f1c74ca14e9a16dda22f66f78